src/loader.js

```diff
@@ -284,6 +284,85 @@ function setupModuleLoader(window) {
 
           /**
            * @ngdoc method
+           * @name angular.Module#component
+           * @module ng
+           * @param {string} name Name of the component in camel-case (i.e. myComp which will match as my-comp)
+           * @param {Object} options Component definition object, has the following properties (all optional):
+           *
+           *    - `controller` – `{(string|function()=}` – Controller fn that should be associated with
+           *      newly created scope or the name of a {@link angular.Module#controller registered
```

Comment:
Perhaps a link to the controller section in `$compile`: https://docs.angularjs.org/api/ng/service/$compile#-controller